authors/apps/articles/urls.py

```diff
@@ -2,14 +2,17 @@
 from .views import (
     ListCreateArticleAPIView,
     RetrieveUpdateArticleAPIView,
-    FetchTags
-)
+    FetchTags,
+    LikeDislikeView)
+from django.conf.urls import url
 
 app_name = 'articles'
 
 urlpatterns = [
     path('articles/', ListCreateArticleAPIView.as_view(), name='article'),
     path('articles/<slug>/', RetrieveUpdateArticleAPIView.as_view(),
          name='articles'),
-    path('tags/', FetchTags.as_view(), name="all_tags")
+    path('tags/', FetchTags.as_view(), name="all_tags"),
+    path('articles/<str:slug>/<str:vote_type>',
```

Comment:
can you append a `/` to this URL pattern to ensure it uses the same pattern as the others.

Reply:
If I append the  ```/``` my get method clashes with the ```path('articles/<slug>/', RetrieveUpdateArticleAPIView.as_view(),
         name='articles'),```